tf_encrypted/convert/convert.py

```diff
@@ -279,21 +286,30 @@ def find_leaves(scope, subscope_map):
   return input_leaves, output_leaves
 
 
-def match_numbered_scope(specop, search_string, return_group=True):
+def match_numbered_scope(specop, search_string,
+                         return_group=True,
+                         not_numbered=False):
   """
   Find a numbered scope matching a specop from REGISTERED_SPECOPS,
   and return it if found
 
+  :param not_numbered: only match exact numbering  -
```

Comment:
Hey, we're actually moving toward [Google Style Docstrings](http://google.github.io/styleguide/pyguide.html#383-functions-and-methods)! Sorry for the confusion

Reply:
@jvmancuso changed for the ```numbered``` param but wasn't sure what to write about the ```return_group``` param.

Reply:
fixed, added some definitions in the docstring